src/marketing/email-template-message.md

```diff
@@ -52,7 +52,13 @@ The process of customizing the body of each message is the same as for customizi
 
    `customVar code=my_custom_variable`
 
-1. To make CSS declarations, enter the styles in **Template Styles**.
+1. To make CSS declarations, enter the styles in **Template Styles**.
+
+   ![Email templates - add custom styles]({% link images/images/email-template-add-custom-styles.png %}){: .zoom}
+   _Add Custom Styles_
+
+   {:.bs-callout-info}
+   Custom styles will be applied to email if `{{template config_path="design/email/header_template"}}` is present in the **Template Styles** otherwise does not. To use custom CSS without a default header template you need to provide them in the **Template Styles** in the `<style>` HTML tag.
```

Comment:
```suggestion
   Custom styles are applied to the email only if `{{template config_path="design/email/header_template"}}` is present in the **Template Styles** . To use custom CSS without a default header template, you must provide them here within the `<style>` HTML tag.
```

Reply:
Hi, @leslietilling !

Your suggestion has been applied. 

Could you please review?

Thank you!